### DISCUSSION TITLE
Meetup Membership

### DISCUSSION THREAD

**Discussion by mmisztal1980** [dotnet-foundation/Home#86] [General Discussions]
Hello,

I'm one of the leaders of the Warsaw .NET Usergroup (#wgdotnet). We're one of the oldest .NET meetups in Poland (we've been around since ~2006 if memory serves me correctly).

During this time, the meetup appears to have joined the dotnet-foundation, which is indicated by the dotnet-foundation logo on our [meetup.com](https://www.meetup.com/WG-NET/) page

Unfortunately the context of this decision, and any side-effects caused by this are unclear to us at this time (the old leadership didn't provide us with any knowledge on this topic), therefore I'd like to ask:

- Is our meetup associated with the dotnet-foundation? What do your records say?
- What is the dotnet foundation membership for meetups associated with? Are there any benefits or duties we should be aware of?

Best regrads,
Maciej

**Comment by nicoleabuhakmeh** [comment]
Hi there! 

I do have the [Warsaw .NET Usergroup](https://www.meetup.com/WG-NET/) meetup group listed within our network and you are listed as one of the organizers. I don't have any specifics of when the group was added, but I can see the date it was created as 11/12/14. 

We welcome .NET related meetups that want to be a part of our network and we cover the associated cost of the groups within our network. We try to maintain the health of the network by asking groups to keep events scheduled. We evaluate our groups quarterly for Meetups that haven't had meetups within the last 6 months or have anything planned going forward.

**Comment by mmisztal1980** [comment]
Hello again,

Ok, glad that's verified :) I think we've been steadily satisfying those criteria (even during Covid). 

Does the dotnet-foundation support the associated meetups in any way? SWAG for attendees/speakers? meetup.com subscriptions? Networking to invite some foreign speakers?

**Reply by mmisztal1980** [reply]
@nicoleabuhakmeh can you provide some guidance regarding the above, please? :)

**Reply by nicoleabuhakmeh** [reply]
Hi again... 

I have some swag items I can get out to you, can you send me an email to nicole.miller@dotnetfoundation.org with your postal address? 

We cover the subscription for the group, but not individuals.  

Also, please check out the [Speaker Directory](https://dotnetfoundation.org/community/speakers) for contacting individuals to speak at your events. 

Thank you!

**Reply by mmisztal1980** [reply]
I'll contact you via email :)

**Reply by mmisztal1980** [reply]
Hi @nicoleabuhakmeh , have you received an email from me?

**Comment by mmisztal1980** [comment]
@nicoleabuhakmeh did you receive an email from me a while back?
